driver/modern_bpf/programs/attached/events/sched_process_exec.bpf.c

```diff
@@ -197,6 +197,20 @@ int BPF_PROG(t1_sched_p_exec,
 	u64 cap_effective = extract__capability(task, CAP_EFFECTIVE);
 	auxmap__store_u64_param(auxmap, cap_effective);
 
+	/* Parameter 24: exe_file ino (type: PT_UINT64) */
```

Comment:
we need to add all these params also in the modern BPF tests, this is the reason why they are failing they expect only `23` params but now they are `26`. BTW tests could be useful to check if the logic is working as we expect

Reply:
Same for all other events